spec/round_spec.rb

```diff
@@ -0,0 +1,74 @@
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
+require './lib/round'
+
+card_1 = Card.new("What is my name?", "Rontron", :People)
```

Comment:
Same with var declaration -- put them in the Rspec describe

Reply:
Encapsulation!